src/status_im2/common/standard_auth/view.cljs

```diff
@@ -0,0 +1,174 @@
+(ns status-im2.common.standard-auth.view
+  (:require [react-native.core :as rn]
+            [quo2.components.numbered-keyboard.numbered-keyboard.view :as numbered-keyboard]
+            [quo2.core :as quo]
+            [status-im2.common.standard-auth.style :as style]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [react-native.fast-image :as fast-image]
+            [status-im2.common.resources :as resources]
+            [utils.i18n :as i18n]
+            [quo2.components.keycard.view :as keycard]
+            [utils.re-frame :as rf]))
+
+(defn enter-keycard-pin-sheet
+  []
+  (let [entered-numbers (reagent/atom [])
+        max-digits 6
+        max-attempt-reached (reagent/atom false)
+
+        handle-number-entry
+        (fn [number]
```

Comment:
I would consider moving this to a defn outside the component and pass in the values.

Reply:
@J-Son89 Moved `handle-number-entry` outside the component.